### PR TITLE
[ISSUE-125] Update operator documentation

### PR DESCRIPTION
**Pull request by AnnaDutova**
Signed-off-by: Dutova <Ann.Dutova@Dell.com>

### Issue #125 

Due to proposal dell/csi-baremetal#806 need to add steps for manual patching scheduler on k3s

## PR checklist
- [x] Add link to the issue
- [x] Choose Project
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
Was already tested in  dell/csi-baremetal#806

### CODE REVIEW

File: docs/MANUAL_SCHEDULER_CONFIGURATION.md [hunk 1]
```diff
@@ -127,6 +127,44 @@ following manual steps are required. Note that user must wait for all scheduler
     * Follow instructions for vanilla Kubernetes, but use the following path to the scheduler configuration file
     `/var/lib/rancher/rke2/agent/pod-manifests/`
 
+* K3S
+    * For manual patching folow next instructions
```
Comment (safronovD) [inline docs/MANUAL_SCHEDULER_CONFIGURATION.md:131]: typo _follow_ and _the instructions below_ is better

Comment (safronovD) [inline docs/MANUAL_SCHEDULER_CONFIGURATION.md:131]: Please add _for all master nodes_

**Review by misanthropicat — APPROVED** [review]

File: docs/MANUAL_SCHEDULER_CONFIGURATION.md [hunk 2]
```diff
@@ -127,6 +127,44 @@ following manual steps are required. Note that user must wait for all scheduler
     * Follow instructions for vanilla Kubernetes, but use the following path to the scheduler configuration file
     `/var/lib/rancher/rke2/agent/pod-manifests/`
 
+* K3S
+    * For manual patching folow next instructions  
+      * Create next configuration file in directory `/var/lib/rancher/k3s/agent/pod-manifests/scheduler`:
+        
+        ```
```
Comment (safronovD) [inline docs/MANUAL_SCHEDULER_CONFIGURATION.md:134]: `yaml` here can help to colorize listings for popular MD files viewers

File: docs/MANUAL_SCHEDULER_CONFIGURATION.md [hunk 3]
```diff
@@ -127,6 +127,44 @@ following manual steps are required. Note that user must wait for all scheduler
...
+        apiVersion: kubescheduler.config.k8s.io/v1beta1
```
Comment (safronovD) [inline docs/MANUAL_SCHEDULER_CONFIGURATION.md:135]: Please add that we should use `kubescheduler.config.k8s.io/v1beta3` for k8s 1.23 and above

**Review by safronovD — APPROVED** [review]

File: docs/MANUAL_SCHEDULER_CONFIGURATION.md [hunk 4]
```diff
@@ -127,6 +127,46 @@ following manual steps are required. Note that user must wait for all scheduler
     * Follow instructions for vanilla Kubernetes, but use the following path to the scheduler configuration file
     `/var/lib/rancher/rke2/agent/pod-manifests/`
 
+* K3S
```
Comment (mishoyama) [inline docs/MANUAL_SCHEDULER_CONFIGURATION.md:130]: Please add reference on this to main README.md

**Review by mishoyama — CHANGES_REQUESTED** [review]

**Review by mishoyama — APPROVED** [review]

**Pull request merged**